hub/core/meta/tensor_meta.py

```diff
@@ -35,6 +36,7 @@ def default_tensor_meta(
     tensor_meta["chunk_size"] = chunk_size
     tensor_meta["dtype"] = dtype
     tensor_meta["length"] = 0
+    tensor_meta["version"] = hub.__version__
```

Comment:
What's the need to have meta on a tensor level?

Reply:
yes :-P

Reply:
better safe than sorry

Reply:
index map should also have it, but i'll throw that in backlog

Reply:
AL-1067 if interested